docs/source/setup/index.rst

```diff
@@ -1,69 +1,140 @@
 .. _setup:
-===================
-Setting up VOLTTRON
-===================
 
-Installation
-------------
+.. _Building-VOLTTRON:
 
-:ref:`Install VOLTTRON <install>` by running the following commands that installs needed
-:ref:`prerequisites <VOLTTRON-Prerequisites>`, clones the source code, then builds the virtual environment for using the platform.
+Building VOLTTRON
+=================
 
-.. code-block:: bash
+The VOLTTRON project includes a bootstrap script which automatically
+downloads dependencies and builds VOLTTRON. The script also creates a
+Python virtual environment for use by the project which can be activated
+after bootstrapping with ". env/bin/activate". This activated Python
+virtual environment should be used for subsequent bootstraps whenever
+there are significant changes. The system's Python need only be used on
+the initial bootstrap.
 
-    sudo apt-get update
-    sudo apt-get install build-essential python-dev openssl libssl-dev libevent-dev git
-    git clone https://github.com/VOLTTRON/volttron
+Install Required Software
```

Comment:
Proper RST only has the length of the title/section.  This will cause less warnings during the running of make html.